src/platform/graphics/constants.js

```diff
@@ -1020,9 +1119,21 @@ export const isIntegerPixelFormat = (format) => {
     return pixelFormatInfo.get(format)?.isInt === true;
 };
 
+/**
+ * Returns the linear format for the given sRGB format. If it does not exist, the input format is
+ * returned. For example for {@link PIXELFORMAT_RGBA8} the return value is {@link PIXELFORMAT_RGBA8}.
+ *
+ * @param {number} format - The texture format.
+ * @returns {number} The linear format for the given sRGB format.
+ * @ignore
+ */
+export const getPixelFormatLinearMatch = (format) => {
```

Comment:
Since this function maps linear formats to srgb, it could probably have a better name, maybe something like `mapFormatLinearToSrgb`?